udl-list.json

```diff
@@ -37,6 +37,2090 @@
+			"version": "Sun, 05 Aug 2012 13:34:32 GMT",
+			"repository": "",
+			"description": "SAP Web Intelligence Formula Editor",
+			"author": "Ivan Rubil <mailto:rubil.ivan@gmail.com>"
+		},
+		{
+			"id-name": "SAS_byCathLawrence",
+			"display-name": "SAS",
+			"version": "Thu, 04 Nov 2010 18:32:55 GMT",
+			"repository": "",
+			"description": "SAS",
+			"author": "Cath Lawrence <mailto:cath.lawrence@aihw.gov.au>"
+		},
+		{
+			"id-name": "SAS_withPseudocode_byJeffFoarde",
+			"display-name": "SAS with pseudocode",
+			"version": "Sat, 09 Feb 2013 13:32:55 GMT",
+			"repository": "",
+			"description": "SAS with pseudocode",
+			"author": "Jeff Foarde <mailto:foarde@gmail.com>"
+		},
+		{
+			"id-name": "SchlumbergerEclipse-slb_by-juanbretti",
+			"display-name": "Schlumberger Eclipse",
+			"version": "Thu, 15 Apr 2010 17:22:35 GMT",
+			"repository": "",
+			"description": "Schlumberger Eclipse",
+			"author": "Juan Pedro Bretti Mandarano <http://www.google.com/profiles/juanbretti>"
+		},
+		{
+			"id-name": "ShadeLab_by-redcool007",
+			"display-name": "ShadeLab",
+			"version": "Fri, 16 Aug 2013 13:46:37 GMT",
+			"repository": "",
+			"description": "ShadeLab",
+			"author": "<mailto:redcool007@foxmail.com>"
+		},
+		{
+			"id-name": "Siebel_eScript_byGiuseppeGarofalo",
+			"display-name": "Siebel eScript",
+			"version": "Mon, 12 Sep 2011 06:57:13 GMT",
+			"repository": "",
+			"description": "Siebel eScript",
+			"author": "Giuseppe Garofalo <mailto:giupino.garofalo@gmail.com>"
+		},
+		{
+			"id-name": "Siemens-SCL_byTizianoSartini",
+			"display-name": "Siemens SCL (definition and autocompletion)",
+			"version": "Sat, 16 Mar 2013 12:05:43 GMT",
+			"repository": "",
+			"description": "Siemens SCL (definition and autocompletion)",
+			"author": "Tiziano Sartini <mailto:tiziano.sartini@gmail.com>"
+		},
+		{
+			"id-name": "SLAX_byTravisWaltz",
+			"display-name": "SLAX",
+			"version": "Wed, 21 Aug 2013 21:23:57 GMT",
+			"repository": "",
+			"description": "SLAX",
+			"author": "Travis Waltz <mailto:tcw135@psu.edu>"
+		},
+		{
+			"id-name": "SmallWorldMagik_byChristianTaton",
+			"display-name": "SmallWorld Magik",
+			"version": "Wed, 14 Jul 2010 08:05:28 GMT",
+			"repository": "",
+			"description": "SmallWorld Magik",
+			"author": "Christian Taton <mailto:tatonchristian@gmail.com>"
+		},
+		{
+			"id-name": "Smarty_byHelge-deVries",
+			"display-name": "Smarty",
+			"version": "Wed, 03 Aug 2005 19:42:49 GMT",
+			"repository": "",
+			"description": "Smarty",
+			"author": "Helge de Vries <mailto:royan@rootpower.net>"
+		},
+		{
+			"id-name": "Snort_by-tropismgroup",
+			"display-name": "Snort",
+			"version": "Sat, 11 Aug 2012 12:04:24 GMT",
+			"repository": "",
+			"description": "Snort",
+			"author": "http://www.tropismgroup.org/2012/08/02/snort-user-defined-language-udl-in-notepad/ <http://www.tropismgroup.org/2012/08/02/snort-user-defined-language-udl-in-notepad/>"
```

Comment:
The content of URL is not readable. 
Let's put `Anonymous`, and remove the URL for the moment.

Reply:
URL removed.  Since I had called it `Snort_by-tropismgroup`, I decided to use same identifier as the author.
https://github.com/pryrt/userDefinedLanguages/commit/6f8f59e85f21add32590c3d3f318852156db7cc5

Reply:
I don't even think `tropismgroup` it's accurate.
http://www.tropismgroup.org/ is a lottery website and people need to login for some operation.
So let's put Anonymous until people come to claim it.

Reply:
https://github.com/pryrt/userDefinedLanguages/commit/e70fd6cce6bb35581fffb58806560432af86475d switched to anonymous